### ISSUE TITLE
Unable to list / directory on object storage system (Listing directory / failed)

### ISSUE THREAD

**Issue by rushabhpasad** [iterate-ch/cyberduck#14550]
@dkocher @ylangisc 
I am using Cyberduck to connect to on premise object storage system. With the release of v8.4.3, I am unable to list the buckets on a particular file system. After going through the change log, the issue seems to be with PR #13664. This PR makes bucket name to be a mandatory field and disallows the listing of all buckets. Can this issue be addressed?
<img width="256" alt="Screenshot 2023-04-19 at 10 43 49" src="https://user-images.githubusercontent.com/1585077/233157092-99caeafc-bdab-4e45-a8ca-d4184bd3cf0b.png">

**Comment by dkocher** [comment]
Can you provide information about the S3 server implementation and a HTTP transcript [^1].

[^1]: https://docs.cyberduck.io/cyberduck/support/#logging-output

**Comment by rushabhpasad** [comment]
I do not have the information for the s3 server implementation.

Attached is the log file containing the exception and relevant details. Sensitive information has been redacted.

[cyberduck.log](https://github.com/iterate-ch/cyberduck/files/11289142/cyberduck.log)

**Comment by dkocher** [comment]
The log confirms that the `GET /` to list all buckets is returned with a `401` error indicating a problem with the signature for the request sent in the `Authorization` header. For listing buckets the default region `us-east-1` is used as no region can be derived from a bucket. @rushabhpasad Can you compare the `Authorization` header sent between the [8.4.2](https://trac.cyberduck.io/milestone/8.4.2) and the current release?

If the region in the AWS4 request signature is the problem, this could probably be resolved by creating a custom [connection profile](https://docs.cyberduck.io/protocols/profiles/) with a preconfigured default region like

```xml
    <key>Region</key>
    <string>us-phoenix-1</string>
```

See [^1] for examples.

[^1]: https://github.com/iterate-ch/profiles

**Comment by dkocher** [comment]
> I do not have the information for the s3 server implementation.

The response shows `Server: dlb/1.0.2`. Would be interested to know what S3 implementation that refers to.

**Comment by rushabhpasad** [comment]
I don't believe there is any region associated with our implementation. Attaching the logs from v8.4.2 which doesn't throw the exception.
[cyberduck-8.4.2.log](https://github.com/iterate-ch/cyberduck/files/11290445/cyberduck-8.4.2.log)

**Comment by dkocher** [comment]
We obfuscate the `Authorization` in the transcript, thus I cannot tell the difference. @rushabhpasad You will have to inspect using a proxy such as [^1].

[^1]: https://proxyman.io/
